esx_service/vmdk_ops.py

```diff
@@ -1703,6 +1704,101 @@ def wait_for_tasks(si, tasks):
 
 #------------------------
 
+def start_vm_changelistener():
+    """
```

Comment:
As of now, the thread is in daemon mode and it stops with the process is restarted.
Had this not been a daemon, the main thread before exiting, waits for this listener to complete( it is waiting for update or updating the vmdk status).